### PR TITLE
Add link for article exception

### PR DESCRIPTION
**Pull request by lepiaf**

### CODE REVIEW

**Comment by ElevenWilson** [comment]
29 words might be misspelled, please check them: amoutte, débogage, encapsuler, Encapsuler, eval, generatePassword, gerer, getMyObject, getPrevious, lang, l'encapsulation, LogicException, loguer, MyClass, MyObject, MyObjectException, ParseError, parsez, passwordGenerator, personalisée, rethrow, RuntimeException, throwable, Throwable, TypeError, typehint, UpdateContentException, url, UserFactoryException

**Comment by ElevenWilson** [comment]
65 words might be misspelled, please check them: amoutte, ArgumentCountError, ArithmeticError, AssertionError, backtrace, BadFunctionCallException, BadMethodCallException, chaine, ClosedGeneratorException, developpeur, DivisionByZeroError, DomainException, DOMException, ErrorException, eval, generatePassword, getCode, getFile, getLine, getMessage, getMyObject, getPrevious, getTrace, getTraceAsString, IntlException, InvalidArgumentException, lang, laquel, LengthException, LoggerAwareInterface, LoggerAwareTrait, LogicException, MyClass, myCustomMethod, MyException, myObject, MyObject, MyObjectException, MyThrowable, NullLogger, OutOfBoundsException, OutOfRangeException, OverflowException, ParseError, passwordGenerator, PasswordGeneratorInterface, PDOException, PharException, Psr, RangeException, rééssayer, ReflectionException, RuntimeException, setPassword, throwable, Throwable, toString, TypeError, ulterieurement, UnderflowException, UnexpectedValueException, UpdateContentException, UserFactory, UserFactoryException, variadic

**Comment by instabledesign** [comment]
@titi et moi tu ajoute pas mon lien? 😞

**Comment by lepiaf** [comment]
quel lien?

**Comment by instabledesign** [comment]
bah le lien de mon article (cross link)

File: _posts/en/2018-02-11-throwable-error-exception.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: post
 title: PHP 7 Throwable Errors Exceptions 
-lang: fr
+lang: en
```
Comment (lepiaf) [inline _posts/en/2018-02-11-throwable-error-exception.md]: haha mauvais copité collé. le lien est foireux maintenant. 

https://blog.eleven-labs.com/en/fr/php7-throwable-error-exception/

@instabledesign

**Comment by ElevenWilson** [comment]
94 words might be misspelled, please check them: actionType, amoutte, ArgumentCountError, ArithmeticError, AssertionError, backtrace, BadFunctionCallException, BadMethodCallException, chaine, ClosedGeneratorException, CouldNotCopyFileException, developpeur, DivisionByZeroError, DivisionByZeroException, DomainException, DOMException, errorCode, ErrorException, errorNames, eval, EventListener, ExceptionInterface, ExceptionListener, FileNotFoundException, Filesystem, generatePassword, getCode, getElement, getErrorName, getFile, getLine, getMessage, getMyObject, getPrevious, getResponse, getStatusCode, getTrace, getTraceAsString, HttpKernel, IntlException, InvalidArgumentException, isset, lang, laquel, LengthException, LoggerAwareInterface, LoggerAwareTrait, LogicException, MyClass, myCustomMethod, MyException, myObject, MyObject, MyObjectException, MyThrowable, NullLogger, OperationNotPossibleException, originFile, OutOfBoundsException, OutOfRangeException, OverflowException, overwriteNewerFiles, ParseError, passwordGenerator, PasswordGeneratorInterface, PDOException, PharException, PropertyPath, Psr, RangeException, rééssayer, ReflectionException, RequestException, runtime, RuntimeException, setPassword, SPL, sprintf, src, Symfony, targetFile, throwable, Throwable, toString, tthuon, TypeError, ulterieurement, UnderflowException, UnexpectedValueException, UpdateContentException, UserFactory, UserFactoryException, username, variadic

**Pull request merged**